blockchain/scriptval.go

```diff
@@ -83,15 +86,19 @@ out:
 
 			// Create a new script engine for the script pair.
 			sigScript := txIn.SignatureScript
+			witness := txIn.Witness
+			inputAmount := txEntry.AmountByIndex(originTxIndex)
 			vm, err := txscript.NewEngine(pkScript, txVI.tx.MsgTx(),
-				txVI.txInIndex, v.flags, v.sigCache)
+				txVI.txInIndex, v.flags, v.sigCache, txVI.sigHashes,
+				inputAmount)
 			if err != nil {
 				str := fmt.Sprintf("failed to parse input "+
 					"%s:%d which references output %s:%d - "+
-					"%v (input script bytes %x, prev output "+
-					"script bytes %x)", txVI.tx.Hash(),
-					txVI.txInIndex, originTxHash,
-					originTxIndex, err, sigScript, pkScript)
+					"%v (input witness %v, input script "+
```

Comment:
Switched to use `%x` instead which will recursively display all items using a hex encoding.